pages/login.tsx

```diff
@@ -57,7 +62,7 @@ export default function Login(): JSX.Element {
                     <FormControl id="email">
```

Comment:
I think some of the text should be center-aligned: 


For line 52
```typescript
                        <Text
                            fontWeight="400"
                            fontSize="16px"
                            mt="20px"
                            textAlign="center"
                        >
                            Registration for Summer 2021 classes begin June 31,
                            2021
                        </Text>
```

![image](https://user-images.githubusercontent.com/44826218/142134397-73ce42bb-b438-4362-928b-7a4e1ef11132.png)